RecoHGCal/TICL/plugins/TrackstersPCA.h

```diff
@@ -3,8 +3,6 @@
 
 #include "DataFormats/HGCalReco/interface/Trackster.h"
 #include "DataFormats/CaloRecHit/interface/CaloCluster.h"
-#include "DataFormats/Common/interface/ValueMap.h"
```

Comment:
I think this ValueMap.h is still required to stay here (see the third argument in `assignPCAtoTracksters`) ...

Reply:
@perrotta it compiles without that `include`, shall I add it back?

Reply:
> @perrotta it compiles without that `include`, shall I add it back?

It remains a mystery to me why it requires the include in the implementation file, but not in the header... But ok,if it works as such don't revert it.
You can however remove `#include "TPrincipal.h"` from the implementation file: that's certainly not needed there.

Reply:
Done in the latest commit.